python/BUILD

```diff
@@ -1,11 +1,18 @@
-load("@rules_python//python:defs.bzl", "py_library")
 load("@pgv_pip_deps//:requirements.bzl", "all_requirements")
+load("@rules_python//python:defs.bzl", "py_library")
+load("@rules_python//python:pip.bzl", "compile_pip_requirements")
 
 exports_files([
+    "requirements.txt",
     "requirements.in",
     "setup.cfg",
 ])
 
+compile_pip_requirements(
```

Comment:
this also creates a test to make sure the lockfile is updated, so the new lockfile is slightly easier to manage with that